proposals/1-embedding_retriever.md

```diff
@@ -0,0 +1,118 @@
+- Start Date: 2022-11-11
```

Comment:
Can you move the file under `text/3558-embedding_retriever.md`?

Reply:
Aha, I see 👍